src/routes/formsgSiteLaunch.ts

```diff
@@ -62,8 +57,25 @@ interface RedirectionDomainObject {
   target: string
 }
 
+interface LaunchFailureEmailProps {
+  requesterEmail?: string
+  repoName?: string
+  primaryDomain?: string
+  error: string
+}
+
+interface DnsRecordsEmailProps {
+  requesterEmail: string
+  repoName: string
+  domainValidationSource: string
+  domainValidationTarget: string
+  primaryDomainSource: string
+  primaryDomainTarget: string
+  redirectionDomain?: RedirectionDomainObject
+}
+
 export class FormsgSiteLaunchRouter {
-  siteLaunch = async (formResponses: FormResponsesProps): Promise<void> => {
+  siteLaunch = async (formResponses: FormResponsesProps) => {
```

Comment:
I am not very happy with the naming of this function `siteLaunch` since there also exists another function called `siteLaunchHandler`, but I am not too sure how to improve this, open to suggestions!

Reply:
the method `launchSiteUsingForm` also exists :sadge:

Reply:
names are too similar gg @__@ but can't really think of a better naem too